catalog/plugin/catalog-plugin-oauth/src/main/java/org/codice/ddf/catalog/plugin/oauth/OAuthPlugin.java

```diff
@@ -140,17 +141,22 @@ public QueryRequest process(Source source, QueryRequest input) throws StopProces
 
     Object securityAssertion = input.getProperties().get(SECURITY_SUBJECT);
     if (!(securityAssertion instanceof Subject)) {
-      LOGGER.warn("A user Subject is not available.");
-      throw new StopProcessingException("A user Subject is not available.");
+      LOGGER.warn("The user's subject is not available.");
```

Comment:
✏️ I feel like I've mentioned this before, sorry if I did, but would "The user Subject is not available."  make more sense here?